src/Components/Web.JS/src/Rendering/LogicalElements.ts

```diff
@@ -52,15 +52,13 @@ export function toLogicalRootCommentElement(start: Comment, end: Comment): Logic
 
   const parent = start.parentNode;
   const parentLogicalElement = toLogicalElement(parent, /* allow existing contents */ true);
-  const children = getLogicalChildrenArray(parentLogicalElement);
-  Array.from(parent.childNodes).forEach(n => children.push(n as unknown as LogicalElement));
```

Comment:
is this just cleanup? I don't see `children` used anywhere.

Reply:
This logic is populating the "logical children array" with the physical children during init. It's needed so the logical element is correctly initialized. The `children` variable here is not used later, but the array it references might be.

Previously it was handled as a standalone special case, but I changed this to be part of the `toLogicalElement` since that's the shared initialization code path.

Reply:
Update: no longer changing this. It's simpler to leave it as it was.